packages/commonwealth/client/scripts/views/pages/discussions/ThreadCard/ThreadCard.tsx

```diff
@@ -209,7 +209,7 @@ export const ThreadCard = ({
             <CWText type="b1" className="content-body">
               <QuillRenderer
                 doc={bodyText}
-                cutoffLines={4}
+                maxChars={500}
```

Comment:
Can we maybe put this as a constant in `shared/src/constants.ts`? It is a useful reference for me on the backend where we will soon be truncating all bodies longer than the amount required for previews.

Reply:
sure , i will put there . 
"I noticed that we're using <QuillRenderer /> in different places, and in some instances, we're setting cutoffLines to 50 or 10. Should I replace all occurrences of cutoffLines with maxchars, or should we keep support for cutoffLines so that it continues to work as it currently does in other places?"
@Israellund

Reply:
@KaleemNeslit I think this is a good question for @mzparacha What do you think?

Reply:
should i add 5000 character for 50 cutoffLines and 1000 character for 10 lines @Israellund ?

Reply:
@KaleemNeslit See above comment by Sachin